test/bdd/agent_steps.go

```diff
@@ -14,14 +14,19 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hyperledger/aries-framework-go/pkg/framework/didresolver"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didmethod/httpbinding"
+
```

Comment:
import orders should be `build-in, third-party and local`

Reply:
done